content/en/docs/Tasks/adding-an-admission-webhook.md

```diff
@@ -0,0 +1,83 @@
+# Admission Webhooks
+
+## Background
+
+OLM is capable of managing the lifecycle of [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks that are shipped alongside your operator.
```

Comment:
This paragraph seems like a good candidate for the description section. See https://github.com/operator-framework/olm-docs/blob/master/content/en/docs/Tasks/install-operator-with-olm.md for example.

Reply:
I'm not a fan of how it is rendered.